client/src/Commands/Commands.js

```diff
@@ -0,0 +1,101 @@
+import React from 'react'
+import styled from 'styled-components'
+import { Icon } from '@common/Icon'
+
+const Container = styled.div`
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  max-width: 50%;
+  border: 1px solid grey;
+  padding: 20px;
+  margin: 20px;
+  border-radius: 8px;
+`
+
+const List = styled.li`
+  list-style-type: none;
+  border: 1px solid grey;
+  padding: 16px;
+  width: 100%;
+  border-radius: 8px;
+  display: flex;
+  justify-content: space-between;
+  width: 90%;
+  &:not(:first-child) {
+    margin-bottom: 16px;
+  }
+  span {
+    color: grey;
+  }
+`
+
+const Controls = styled.div`
+  display: flex;
+  justify-content: space-between;
+  gap: 8px;
+  svg.CHEVRON {
+    transform: rotate(180deg);
+    path {
+      fill: grey;
+    }
+  }
+`
+
+const SubContainer = ({ label, description, children }) => {
```

Comment:
Maybe we don't need this component? none of the props are being used except for children.  Consider just returning `<List>...</List>` where you need this element?